plugins/scaffolder-backend/src/scaffolder/tasks/StorageTaskBroker.test.ts

```diff
@@ -243,8 +244,10 @@ describe('StorageTaskBroker', () => {
 
     const task = await storage.getTask(taskId);
 
-    const promise = broker.list({ createdBy: 'user:default/foo' });
-    await expect(promise).resolves.toEqual({ tasks: [task] });
+    const promise = broker.list({
+      filters: { createdBy: ['user:default/foo'] },
+    });
+    await expect(promise).resolves.toEqual({ tasks: [task], totalTasks: 1 });
```

Comment:
Would be nice to have a test for each new introduce parameter.
And some tests with a combination of them.

This test proves only that `createdBy` should work 😅

Reply:
The other parameters are tested in other tests but sure I can add more explicit tests